### PR TITLE
initial additions

### PR DESCRIPTION
**Pull request by smutt**
Lots of changes throughout the document.

### CODE REVIEW

File: draft-davies-internal-tld.mkd [hunk 1]
```diff
@@ -33,51 +37,80 @@ informative:
...
     date: 2024-01
+  ICANN-Board-Resolution:
+    title: "Reserving .INTERNAL for Private-Use Applications"
+    target: https://www.icann.org/en/board-activities-and-meetings/materials/approved-resolutions-special-meeting-of-the-icann-board-29-07-2024-en#section2.a
+    date: 2024-07
 
 --- abstract
 
-This document describes the reservation of the "internal" top-level domain for
+This document describes the reservation of the ".internal" top-level domain for
```
Comment (kjd) [inline draft-davies-internal-tld.mkd:47]: Note this is not consistent with other RFCs, e.g. most recently RFC 9121, we do not use the dot. While it is arguably a matter of test, elsewhere where it refers to string/label the dot is certain not part of those.

File: draft-davies-internal-tld.mkd [hunk 2]
```diff
@@ -33,51 +37,80 @@ informative:
...
-the broader Domain Name System, such as within closed corporate or home networks. 
+their own domain naming scheme that is not intended to be part of the global
+DNS, such as within closed corporate or home networks. 
 
-Long ago, private-use IP address ranges were set aside {{RFC1918}} for
-a similar purpose. IANA nominally acts as a steward for this address space
-which allows for individual network users to use the specified ranges for
-applications that are not intended to be globally routable.
+Private-use IP address ranges were set aside in {{RFC1918}} for a similar
+purpose. {{ICANN-Board-Resolution}} IANA nominally acts as a steward for this
```
Comment (kjd) [inline draft-davies-internal-tld.mkd]: Not sure why there is a reference to the ICANN Board resolution here, misplaced?

Comment (smutt) [reply draft-davies-internal-tld.mkd]: Yes, this was erroneous. Thanks for deleting.

File: draft-davies-internal-tld.mkd (hunk 1 above)
Comment (smutt) [reply draft-davies-internal-tld.mkd:47]: There's no consistency.

RFC6761, Section 6.2 -> test. / .test / .test.
RFC6762 -> .local.
RFC8375 -> home.arpa.
RFC9121 -> int
RFC9476 -> .alt

**Pull request merged**